src/pkg/projects/netcoreapp/src/localnetcoreapp.override.targets

```diff
@@ -4,39 +4,45 @@
   <Target Name="OverrideRuntimeFilesFromPackageResolve" Condition="'$(CoreCLROverridePath)' != ''" BeforeTargets="GetFilesFromPackageResolve">
     <Error Condition="!Exists('$(CoreCLROverridePath)')" Text="The path provided to CoreCLROverridePath ($(CoreCLROverridePath)) does not exist." />
     <PropertyGroup>
-      <CoreCLRPDBOverridePath Condition="'$(CoreCLRPDBOverridePath)' == '' and Exists('$(CoreCLROverridePath)/PDB')">$(CoreCLROverridePath)/PDB</CoreCLRPDBOverridePath>
+      <CoreCLRCrossTargetComponentFolder Condition="'$(TargetArchitecture)' == 'arm64'">x64</CoreCLRCrossTargetComponentFolder>
```

Comment:
Maybe I'm not quite following the logic here, but won't this cause the build to try to use nonexistent x64 bits when building targeting arm64 on a real arm64 machine? Does this just need a `AND '$(BuildArchitecture)' != 'arm64'` like https://github.com/dotnet/arcade/blob/993af9410c505680b9a260f3bfd79515c936de12/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.dependency.targets#L260 ?

Reply:
Yep you're correct. Good catch!